### PR TITLE
DOCS-1246 - new mobile functions

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?
adds new mobile functions and screenshots

### Motivation
feelings

### Preview

https://docs-staging.datadoghq.com/cswatt/mobile-functions/mobile

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by nmachado26** [comment]
I made 3 changes to this document (https://docs.google.com/document/d/1qgdHEfvBjmXNsCyLc2K4Gejl5-8wllUxhJMFhVR38rc/edit#) to reflect changes to be made here. Two image changes (highlighted in doc), and change to the text in the last two bullet points of the "Siri Shortcuts and Suggestions" section.

Once these changes are made, confirmed that we are OK to merge this PR. Thanks!

File: content/en/mobile/_index.md [hunk 1]
```diff
@@ -48,19 +48,39 @@ On the Dashboards page, you can view and search all of the dashboards that you h
...
+
+## Home Screen Search
+
+{{< img src="mobile/home-screen-search-2.png" alt="Home Screen Search" style="width:40%;">}}
+
+**iOS only**: Within iPhone Search, filter and search for the name of any desired dashboard. Press on a result to open the dashboard view directly on the mobile app, or press the “Search in App” button to open the search query in the in-app Dashboard List page.
+
+## Siri Shortcuts and Suggestions
+
+**iOS Only**: You can create Siri Shortcuts for Datadog dashboards and monitors. To do so, create a shortcut through the Shortcuts App. For a shortcut to be available to create, you must first execute the desired action at least once in the app. For example, to create an “Open Mobile App Screenboard” shortcut, you must first open the Mobile App Screenboard in your mobile app at least once.
```
Comment (nmachado26) [inline content/en/mobile/_index.md:65]: Change to "For example, to create a “Open AWS Overview Dashboard” shortcut, you must first open the AWS Overview Dashboard in your mobile app at least once."

File: content/en/mobile/_index.md [hunk 2]
```diff
@@ -48,19 +48,39 @@ On the Dashboards page, you can view and search all of the dashboards that you h
...
+
+With the shortcut, you can access your dashboards and monitors through three key actions:
+
+* Pin the shortcut as an icon in your home screen. To do so, access the Shortcuts app, and open the edit menu for your dashboard shortcut.
+* Siri Voice: say your shortcut name, such as “Open HAproxy and Bunk” in the image below.
```
Comment (nmachado26) [inline content/en/mobile/_index.md:70]: Change to:
Siri Voice: say your shortcut name, such as “Open AWS Overview”, and Siri will open your dashboard in-app.
Siri suggestions: Siri learns your routine, and suggests dashboard shortcuts when you most need them, through a home or lock-screen banner, iPhone search, or iOS 14 Siri Suggestions widget.

File: content/en/mobile/_index.md [hunk 3]
```diff
@@ -48,19 +48,39 @@ On the Dashboards page, you can view and search all of the dashboards that you h
...
+* Siri suggestions: suggestions for frequently opened dashboards will appear in iPhone Search.
+
+{{< img src="mobile/shortcuts-3.png" alt="Shortcuts">}}
```
Comment (nmachado26) [inline content/en/mobile/_index.md:73]: Changed image. In doc here https://docs.google.com/document/d/1qgdHEfvBjmXNsCyLc2K4Gejl5-8wllUxhJMFhVR38rc/edit

File: content/en/mobile/_index.md [hunk 4]
```diff
@@ -48,19 +48,39 @@ On the Dashboards page, you can view and search all of the dashboards that you h
 
 **Note:** To set up or edit a dashboard, you need to [login to the Datadog browser app][9]. For more information, see [Dashboards][5]. 
 
-<!--- ## Quick Actions  --->
+## Quick Actions
 
-<!--- {{< img src="mobile/quick_actions.png" alt="dashboard page">}}  --->
+{{< img src="mobile/quick-actions-3.png" alt="Quick Actions">}}
```
Comment (nmachado26) [inline content/en/mobile/_index.md:53]: Changed image. in doc https://docs.google.com/document/d/1qgdHEfvBjmXNsCyLc2K4Gejl5-8wllUxhJMFhVR38rc/edit

**Review by nmachado26 — DISMISSED** [review]
Thank you so much! Let me know how I can help

File: content/en/mobile/_index.md (hunk 1 above)
Comment (nmachado26) [reply content/en/mobile/_index.md:65]: The final sentence

**Review by apigirl — APPROVED** [review]

**Pull request merged**